webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/TemplatesCard/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '@patternfly/react-core';
+import { PencilAltIcon } from '@patternfly/react-icons';
+import { TableComposable, Tr, Tbody, Td } from '@patternfly/react-table';
+import { translate as __ } from '../../../../../../common/I18n';
+import { foremanUrl } from '../../../../../../common/helpers';
+import { useAPI } from '../../../../../../common/hooks/API/APIHooks';
+import SkeletonLoader from '../../../../../common/SkeletonLoader';
+import DefaultLoaderEmptyState from '../../../../DetailsCard/DefaultLoaderEmptyState';
+import CardTemplate from '../../../../Templates/CardItem/CardTemplate';
+import { STATUS } from '../../../../../../constants';
+
+const TemplatesCard = ({
+  hostName,
+  hostDetails: { managed },
+  isExpandedGlobal,
+}) => {
+  const templatesUrl = foremanUrl(`/api/hosts/${hostName}/templates`);
+  const TemplateTypeTitle = __('Template Type');
+  const {
+    response: {
+      templates,
+      edit_provisioning_templates: editTemplatePermission,
+    },
+    status,
+  } = useAPI('get', templatesUrl);
+  const editTemplateUrl = id => `/templates/provisioning_templates/${id}/edit`;
+  if (!managed || !templates?.length) return null;
+  return (
+    <CardTemplate
+      header={__('Provisioning templates')}
+      expandable
+      masonryLayout
+      isExpandedGlobal={isExpandedGlobal}
+    >
+      <SkeletonLoader
+        emptyState={<DefaultLoaderEmptyState />}
+        status={status || STATUS.PENDING}
+      >
+        <TableComposable aria-label="templates table" variant="compact">
+          <Tbody>
+            {templates?.map(template => (
+              <Tr key={template.name}>
+                <Td /* to remove padding */ />
+                <Td dataLabel={TemplateTypeTitle} noPadding>
+                  <Button variant="link">{template.name}</Button>
```

Comment:
Until the modal is ready let's just have plain text here, I don't like seeing a link and not being able to click it :cry: